.github/workflows/ci.yaml

```diff
@@ -63,10 +63,6 @@ jobs:
     uses: ./.github/workflows/contracts-deployment-test.yaml
     needs: [ contracts-prettier ]
 
-  contracts-build-and-test:
-    uses: ./.github/workflows/contracts-build-and-test.yaml
-    needs: [contracts-prettier]
-
```

Comment:
@dnkolegov Mind elaborating on why we're removing this workflow?

Reply:
It is the same as contracts-deployment-test. It was renamed and then appeared again probably after merging.

Reply:
Good catch!

Reply:
Actually, not really. They run different make targets in the end.

Reply:
- contracts-deployment.yaml runs `make deploy-ipc`
- contracts-build-and-test.yaml runs `make`

Reply:
Yes, I changed that to run the explicit target